swift_scality_backend/server.py

```diff
@@ -33,6 +33,21 @@
 POLICY_STUB = object()
 
 
+class ScalityDiskFileRouter(object):
```

Comment:
It's related.

Swift's object server has a method `async_update` which access the attribute `_diskfile_router` [1]. This attribute is set up in the `setup` method of the Object server. We completely override thre setup method in our Object Server but we inherit the `async_update` method.

So we need to setup a diskfile router in our setup method, hence this change.

[1] https://github.com/openstack/swift/blob/master/swift/obj/server.py#L254